src/applications/static-pages/vre-chapter31/buildChapter31Content.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import EbenefitsLink from 'platform/site-wide/ebenefits/containers/EbenefitsLink';
+
+export const buildChapter31Content = (page, track) => {
+  let content = '';
+  let optionalHeader = '';
+  let optionalSubwayMapItem = null;
+  // If this is on the /eligibility page, render just the button
+  // If this is NOT on the /eligibility page, render the subway map and the button and possibly the optional header and subway map item
+  if (page === 'eligibility') {
```

Comment:
For values like 'eligibility', 'how-to-apply', 'independent-living', I'd consider storing them in constants instead of declaring them inline.

Reply:
We only needed three of them, I figured creating a variable for this was unnecessary. What would be the benefit?

Reply:
Mm if you could see these values being reused at some point then having them in one place would be beneficial. Not a strong opinion though. If you feel this is fine that's totally cool 👍